tests/unit/preprocessor/_time/test_time.py

```diff
@@ -1022,6 +1022,19 @@ def test_standardized_anomalies(period, standardize=True):
             )
 
 
+@pytest.mark.parametrize('period, reference', PARAMETERS)
+def test_anomalies_preserve_metadata(period, reference, standardize=False):
+    cube = make_map_data(number_years=2)
+    cube.var_name = "si"
+    cube.units = "m"
+    result = anomalies(cube, period, reference, standardize=standardize)
+    assert result.metadata == cube.metadata
```

Comment:
```suggestion
    metadata = copy.deepcopy(cube.metadata)
    result = anomalies(cube, period, reference, standardize=standardize)
    assert result.metadata == metadata
```
or you're only testing whether it's the same, not if it's unchanged

Reply:
:facepalm: cheers :grin: